plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -73,7 +73,10 @@ def __init__(self, repo, publish_conduit, config, distributor_type, association_
         self.rpm_step = PublishRpmStep(dist_step, repo_content_unit_q=association_filters)
         self.add_child(self.rpm_step)
         self.add_child(PublishDrpmStep(dist_step, repo_content_unit_q=association_filters))
-        self.add_child(PublishErrataStep(repo_content_unit_q=association_filters))
+        errata_step = PublishErrataStep()
+        if config.get_boolean(constants.INCREMENTAL_EXPORT_REPOMD_KEYWORD):
+            errata_step = PublishErrataStep(repo_content_unit_q=association_filters)
+        self.add_child(errata_step)
```

Comment:
I would like to use `errata_step = a if some_flag else b` to be honest but I can't...
I will go with your second suggestion then.
Thanks!